.github/workflows/build.yml

```diff
@@ -155,7 +158,10 @@ jobs:
       - name: Install Cygwin
         run: |
           Invoke-WebRequest -UseBasicParsing 'https://cygwin.com/setup-x86_64.exe' -OutFile 'C:\temp\cygwin.exe'
-          Start-Process -Wait -FilePath 'C:\temp\cygwin.exe' -ArgumentList '--packages autoconf,automake,cpio,curl,gcc,git,grep,libtool,make,mingw64-x86_64-gcc-core,perl,pkg-config,unzip,wget,zip --quiet-mode --download --local-install --delete-orphans --site https://mirrors.kernel.org/sourceware/cygwin/ --local-package-dir C:\cygwin_packages --root C:\cygwin64'
+          Invoke-WebRequest -UseBasicParsing 'https://cygwin.com/setup-x86_64.exe.sig' -OutFile 'C:\temp\cygwin.exe.sig'
+          gpg --keyserver keys.gnupg.net --recv-key 676041BA
```

Comment:
Why choosing the old key ? 
>During the transition period, signatures are made using both old (676041BA) and new (1A698DE9E2E56300) public >keys here.

Reply:
hmm I tried with the new key and it failed, looks like the old key failed too

Reply:
@douph1 any idea why the gpg steps are failing?

Reply:
Looks like `keys.gnupg.net` might be down?

```
➜  IcedTea-Web git:(1.8) gpg --recv-key 1A698DE9E2E56300
gpg: keyserver receive failed: No data
➜  IcedTea-Web git:(1.8) ping keys.gnupg.net
ping: cannot resolve keys.gnupg.net: Unknown host
```

Reply:
ahh it needs both...